test/cat.jl

```diff
@@ -316,7 +316,7 @@ end
     df = vcat(DataFrame([CategoricalArray([1])], [:x]),
               DataFrame([CategoricalArray(["1"])], [:x]))
     @test df == DataFrame([[1, "1"]], [:x])
-    @test df[!, :x] isa CategoricalVector{Any}
+    @test df[!, :x] isa CategoricalVector{Union{Int, String}}
```

Comment:
I'd suggest testing a promotion that hasn't changed across CategoricalArrays versions, like `1` and `1.0`.

Reply:
OK - changed